ZeroWasteFriendlyTests/ActivityTiles/ActivityTilesInteractorTests.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ActivityTilesInteractorTests.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 05/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import Quick
+import Nimble
+
+@testable import ZeroWasteFriendly
+
+final class ActivityTilesInteractorTests: QuickSpec {
+
+    override func spec() {
+
+        var router: FakeActivityTilesRouter!
+        var worker: FakeActivityTilesWorker!
+        var presenter: FakeActivityTilesPresenter!
+        var interactor: ActivityTilesInteractorImpl!
+
+        beforeEach {
+            router = FakeActivityTilesRouter()
+            worker = FakeActivityTilesWorker()
+            presenter = FakeActivityTilesPresenter()
+            interactor = ActivityTilesInteractorImpl(presenter: presenter, worker: worker, router: router)
+        }
+
+        describe("") {
```

Comment:
IF you'd have in scene temples these basic flow:

viewDidLoad -interactor.getXXX -worker.fetchXXX - presenter.displayXXX

then you'd be able to test it and make tests to be pregenerated, which is nice